src/ports/items/component.ts

```diff
@@ -26,6 +54,22 @@ export function createItemsComponent(options: {
   }
 
   async function count(filters: ItemFilters) {
+    let option = options.find((option) => option.network === Network.MATIC)
```

Comment:
this logic is repeated in both `fetch` and `count` method, can we abstract it in an independet method so we can re-use it? we could also test it

Reply:
@flobarreto

Reply:
@flobarreto

Reply:
@flobarreto